masterlist.yaml

```diff
@@ -20395,6 +20395,26 @@ plugins:
       - crc: 0x3DC7A6FB
         util: 'TES4Edit v4.0.4c'
 
+  - name: 'FrostcragAlchemySorter.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/13782/'
+        name: 'Frostcrag Alchemy Sorter'
+      - link: 'https://www.nexusmods.com/oblivion/mods/49969/'
+        name: 'Frostcrag Alchemy Sorter Updated'
+    msg:
+      - <<: *obsolete
+        type: error
+        subs: [ '[Frostcrag Alchemy Sorter Updated](https://www.nexusmods.com/oblivion/mods/49969)' ]
+        condition: 'not checksum("FrostcragAlchemySorter.esp", E855599E)'
```

Comment:
The issue with using `not checksum()` is that if the authors of either version ever decide to update their versions, the message would trigger falsely for their users & cause confusion until we update it. I'd prefer to just explicitly check for the checksums of the old mod, even if it's dirty. However, I believe the author is @Sabre142 who often contributes here, so maybe they'd be willing to add version metadata to their version that could be used instead (see `version()` in the [docs](https://loot-api.readthedocs.io/en/stable/metadata/conditions.html#functions)).

Reply:
I have added version info to the ESP's description field.

Reply:
@Sabre142 you are shiva7663? Or Dispensation?

Reply:
I am Dispensation.